concepts/05 Widgets/SelectBox/25 Keyboard Support.md

```diff
@@ -81,6 +81,72 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxSelectBox ...
+            @initialized="registerKeyHandlers"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxSelectBox } from 'devextreme-vue/select-box';
+
+    export default {
+        components: {
+            DxSelectBox
+        },
+        methods: {
+            registerKeyHandlers: function(e) {
+                const selectBoxInstance = e.component;
+                selectBoxInstance.registerKeyHandler("backspace", function (e) {
+                    // The argument "e" contains information on the event
+                });
+                selectBoxInstance.registerKeyHandler("space", function (e) {
+                    // ...
+                });
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import SelectBox from 'devextreme-react/select-box';
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+            this.registerKeyHandlers = this.registerKeyHandlers.bind(this);
```

Comment:
You don't address to the state inside the `registerKeyHandlers` function, so there's no need to bind it to the component's context. You can remove the entire constructor from this example.